typings/link.d.ts

```diff
@@ -68,10 +68,12 @@ export declare interface Sender extends link {
   /**
    * Sends a message
    * @param {Message | Buffer} msg The message to be sent. For default AMQP format msg parameter
-   * should be of type Message interface. For a custom format, the msg parameter should be a Buffer.
+   * should be of type Message interface. For a custom format, the msg parameter should be a Buffer
+   * and a valid value should be passed to the `format` argument.
    * @param {Buffer | string} [tag] The message tag if any.
-   * @param {number} [format] The message format. Usually it is zero. Specify this
-   * if a message with custom format needs to be sent.
+   * @param {number} [format] The message format. Specify this if a message with custom format needs
+   * to be sent. For example, sending multiple messages in a single batch needs the format `0x80013700`.
```

Comment:
Should clarify that 0x80013700 is an Azure/ServiceBus defined batching format. Also rather than 'simple encoded message' we should state that 0 implies the standard AMQP 1.0 defined format.

Reply:
I was wrongly assuming that 0x80013700 was a format defined for AMQP. Removed that entire part.
Also updated what `0` means.
Thanks!

Reply:
Also added what passing nothing to the `format` argument means